docs: Update meta content - HCP and Enterprise (#5284)
hashicorp/boundary: website/content/docs/enterprise/licensing.mdx

layout: docs
page_title: License Boundary Enterprise
description: >-

>   Learn how to enable Boundary Enterprise, once you have a valid license. Request access from an account team to enable enterprise features.

Reviewer: ```suggestion
  Learn how to enable Boundary Enterprise with a valid license. Request access from an account team to enable enterprise features.
```
